src/ametsuchi/tx_store/index.cc

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2016 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <ametsuchi/tx_store/index.h>
+#include <iostream>
+
+namespace ametsuchi {
+namespace tx_store {
+
+using file::offset_t;
+
+Index::Index(const std::string &path) : Index(path, 1024) {}
+
+file::offset_t Index::get(std::size_t n) {
+
```

Comment:
What about using cache here?
And I think `txstore::Array.get(n)` should use cache too.